e2e/tests/functional/plugins/gauge/gauge.e2e.spec.js

```diff
@@ -133,4 +133,50 @@ test.describe('Gauge', () => {
 
     // TODO: Verify changes in the UI
   });
+
+  test('Gauge does not display NaN when data not available', async ({ page }) => {
+    // Create a Gauge
+    await createDomainObjectWithDefaults(page, {
+      type: 'Gauge',
+      name: 'Unnamed Gauge'
+    });
+
+    // Create a Sine Wave Generator in the Gauge with a loading delay
+    await page.getByRole('button', { name: ' Create ' }).click();
+    await page.getByRole('menuitem', { name: ' Sine Wave Generator' }).click();
+    await page
+      .getByRole('dialog')
+      .locator('div')
+      .filter({ hasText: 'Loading Delay (ms)' })
+      .nth(3)
+      .click();
+    await page.getByLabel('Loading Delay (ms)').fill('5000');
+    await page.locator('[aria-label="Save"]').click();
```

Comment:
I actually found a [great example](https://github.com/nasa/openmct/blob/cc8f726bd5de5f83a376283c08b05f96e412de15/e2e/tests/framework/generateLocalStorageData.e2e.spec.js#L142) of adding a load delay to a SWG. The pattern in the link I shared is the recommended way to do this. 

1.  `createExampleTelemetryObject(page, parent)`- creates a default SWG as a child of the parent and returns it
2. Edit the default SWG to apply the loading delay
3. Navigate to the parent object

Reply:
Implemented.